.github/workflows/ci.yml

```diff
@@ -20,7 +20,7 @@
 name: Airflow
 on:
   push:
-    branches: [master]
+    branches: ['*']
```

Comment:
I think this should still be just master or else it would try to run all the tests for commits pushed to v1-10-test too

Reply:
Which is exactly what we want to achieve :)